Dockerfile

```diff
@@ -15,11 +15,12 @@ RUN mv /powershell/PSDepend/0.1.64/PSDependScripts/Noop.ps1 /powershell/PSDepend
 ## base image
 FROM vmware/photon2:20180424
 
-RUN tdnf install -y powershell-6.0.1-1.ph2
+RUN tdnf install -y powershell-6.0.1-1.ph2 gzip tar
```

Comment:
Is gzip required here?

Reply:
Yes. the tarball is compressed with gzip. I'm surprised that installing tar didn't include gzip as a dependency.